src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -403,7 +403,7 @@ private void ImportBasicBlock(BasicBlock basicBlock)
                         break;
                     case ILOpcode.jmp:
```

Comment:
We should actually just throw for that because it's invalid IL and not handling it will result in an unhandled exception. We should at least validate that much. I updated the branch.